src/components/actions/Badge/Badge.stories.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import { fileAbsolute } from 'paths.macro'
-import { Centered } from '../../../storybook/decorators/Centered'
 import { getStoryTitle } from '../../../storybook/get-story-title'
+import { LeftAlign } from '../../../storybook/decorators/LeftAlign'
```

Comment:
Why do some stories required `LeftAlign` while some don't? Wouldn't everything already be left-aligned by default?

Reply:
Since we are in React Native, most of the time we are in a `View`, which by default stretch its children items so they fit the container.